docs/getting-started/transfer-ckbtyes.md

```diff
@@ -0,0 +1,9 @@
+---
+id: transfer-ckbtyes
+title: Transfer CKBytes from Neuron Wallet to the third party
+---
+Please prepare the third party wallet/exchange destination address. **Please make sure your balance is greater than 62 CKB.**
+
+* Under the “Send” tab, fill in the address details in the "Send to" field. Turn on the "Advanced fee settings" and fill in the "Transaction fee" — click the "Send" button to complete your transfer.
```

Comment:
Add description for multi-output transfer by neuron.

Reply:
I don't understand it.

Reply:
update it.